zschema/leaves.py

```diff
@@ -286,22 +292,46 @@ class IPAddress(Leaf):
     ES_TYPE = "ip"
     BQ_TYPE = "STRING"
     EXPECTED_CLASS = [str,unicode]
-    IP_REGEX = re.compile('(\d{1,3}\.){3}\d{1,3}')
+
+    IPV4_REGEX = re.compile('(\d{1,3}\.){3}\d{1,3}')
 
     def _is_ipv4_addr(self, ip):
-        return bool(self.IP_REGEX.match(ip))
+        return bool(self.IPV4_REGEX.match(ip))
+
+    def _is_ipv6_addr(self, ip):
+        try:
+            socket.inet_pton(socket.AF_INET6, ip)
+            return True
+        except socket.error:
+            return False
```

Comment:
It seems like we should use the same methodology for checking IPv4 and IPv6 addresses. Doesn't really matter which to me as long as it's consistent.

Reply:
done in 4a15fc1cf307cf210ee38ffb172f4ed335bc0fb7

Reply:
👍